modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/entity/PodcastEpisode.kt

```diff
@@ -59,6 +59,7 @@ data class PodcastEpisode(
     @ColumnInfo(name = "download_task_id") override var downloadTaskId: String? = null,
     @ColumnInfo(name = "last_archive_interaction_date") var lastArchiveInteraction: Long? = null,
     @ColumnInfo(name = "image_url") var imageUrl: String? = null,
+    @ColumnInfo(name = "deselected_chapters") override var deselectedChapters: ChapterIndices? = ChapterIndices(),
```

Comment:
If we use `ChapterIndicies()` as a default value I think we can make the field not nullable and use an empty string as a default database entry.

Reply:
This worked as well in https://github.com/Automattic/pocket-casts-android/pull/1856/commits/e87c4de8f81eaf7cada328851774b83e02390725